_script/build

```diff
@@ -13,7 +13,7 @@ function binary_compilation {
 	#  https://github.com/kubernetes/kubernetes/pull/26028/files
 	# see also https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html
 	CGO_ENABLED=0 GOOS=linux go build -o out/tcpgoon \
-		    -ldflags '-extldflags "-static"' -a -installsuffix nocgo -tags netgo
+		    -ldflags "-X 'main.version=$(date -u '+0.%Y%m%d%H%M%S')' -extldflags '-static'" -a -installsuffix nocgo -tags netgo
```

Comment:
the idea of hardcoding a "0.minorversion" is good, but move this "0" to a variable. However, I'm a bit reluctant to have the date as the version, and i'd bet by a short version of the commit id (as two builds of the same commit should end up with the same result and the same version)

Reply:
I've gone for something like this:
```
tcpgoon version
Git Commit Hash: d206dc649f0868644d31a1d4849f430b8e9dd851
UTC Build Time: 2017-12-17_03:58:52PM
```
Adding the commit hash plus the build time. good?

Reply:
Yes, but the full commit hash is a bit overkilling, especially if we want to keep it on sync with the docker image tag... what's about?
```bash
 % git rev-parse --short HEAD
8e94bd2
```